phenol-annotations/src/main/java/org/monarchinitiative/phenol/annotations/assoc/GeneInfoLoader.java

```diff
@@ -0,0 +1,52 @@
+package org.monarchinitiative.phenol.annotations.assoc;
+
+import org.monarchinitiative.phenol.annotations.formats.GeneIdentifier;
+import org.monarchinitiative.phenol.annotations.formats.GeneIdentifiers;
+import org.monarchinitiative.phenol.ontology.data.TermId;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.*;
+
+public class GeneInfoLoader {
+  private static final String ENTREZ_GENE_PREFIX = "NCBIGene";
+
+
+  private GeneInfoLoader() {
+    // should not be instantiated
+  }
+
+  /**
+   * @param geneInfoFile Homo_sapiens.gene_info.gz File
+   * @return an immutable Map with keys being NCBI Gene IDs and values being the corresponding symbols
+   */
+  public static GeneIdentifiers loadGeneIdentifiers(Path geneInfoFile) throws IOException {
```

Comment:
Given this has hard-coded the human taxon id as a filter, might it be less surprising for users to name it `GeneInfoLoader.loadHumanGeneIdentifiers` or `HumanGeneInfoLoader.loadGeneIdentifiers`

Reply:
Thanks!